pkg/controller/drain.go

```diff
@@ -154,22 +154,8 @@ func (o *DrainOptions) getController(namespace string, controllerRef *metav1.Own
 	return nil, fmt.Errorf("Unknown controller kind %q", controllerRef.Kind)
 }
 
-func (o *DrainOptions) getPodController(pod api.Pod) (*metav1.OwnerReference, error) {
-	controllerRef := metav1.GetControllerOf(&pod)
-	if controllerRef == nil {
-		return nil, nil
-	}
-
-	// We assume the only reason for an error is because the controller is
-	// gone/missing, not for any other cause.
-	// TODO(mml): something more sophisticated than this
-	// TODO(juntee): determine if it's safe to remove getController(),
-	// so that drain can work for controller types that we don't know about
-	_, err := o.getController(pod.Namespace, controllerRef)
```

Comment:
Do we execute this statement in usual drain scenario?
If not, how does it give any improvement in performance? Does removing this break any other path? Why was this statement there.

Reply:
Yes, this method is called in usual scenarios, making calls to API server. I don't think it breaks any code path if removed.